CRITICAL - 2024-04-27 01:05:48 --> BadMethodCallException: Call to undefined method App\Models\ClienteModel::group_by
in SYSTEMPATH\Model.php on line 874.
 1 APPPATH\Controllers\Cliente.php(64): CodeIgniter\Model->__call('group_by', [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:16:10 --> mysqli_sql_exception: Unknown column 'c.id' in 'field list' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:16:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:16:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:16:10 --> [Caused by] mysqli_sql_exception: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:17:38 --> mysqli_sql_exception: Unknown column 'ciente.personal_id' in 'on clause' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `cliente...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:17:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ciente.personal_id' in 'on clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente`.`id`, `cliente`.`nome` AS `cliente_nome`, `cliente`.`CPF`, `cliente`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:17:38 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ciente.personal_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente`.`id`, `cliente`.`nome` AS `cliente_nome`, `cliente`.`CPF`, `cliente`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente`.`id`, `cliente`.`nome` AS `cliente_nome`, `cliente`.`CPF`, `cliente`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente`.`id`, `cliente`.`nome` AS `cliente_nome`, `cliente`.`CPF`, `cliente`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:17:38 --> [Caused by] mysqli_sql_exception: Unknown column 'ciente.personal_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `cliente`.`id`, `cliente`.`nome` AS `cliente_nome`, `cliente`.`CPF`, `cliente`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente`.`id`, `cliente`.`nome` AS `cliente_nome`, `cliente`.`CPF`, `cliente`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente`.`id`, `cliente`.`nome` AS `cliente_nome`, `cliente`.`CPF`, `cliente`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente`.`id`, `cliente`.`nome` AS `cliente_nome`, `cliente`.`CPF`, `cliente`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:18:08 --> mysqli_sql_exception: Column 'id' in field list is ambiguous in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`, `n...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `n...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `n...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:18:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome` AS `cliente_nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:08 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nome` AS `cliente_nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nome` AS `cliente_nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome` AS `cliente_nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:08 --> [Caused by] mysqli_sql_exception: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`, `nome` AS `cliente_nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nome` AS `cliente_nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nome` AS `cliente_nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome` AS `cliente_nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:18:34 --> mysqli_sql_exception: Column 'id' in field list is ambiguous in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`, `n...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `n...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `n...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:18:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:34 --> [Caused by] mysqli_sql_exception: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`, `nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `CPF`, `email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:18:46 --> mysqli_sql_exception: Unknown column 'c.id' in 'field list' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:18:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:46 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:46 --> [Caused by] mysqli_sql_exception: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:18:46 --> mysqli_sql_exception: Unknown column 'c.id' in 'field list' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:18:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:46 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:46 --> [Caused by] mysqli_sql_exception: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM `cliente`
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:18:49 --> mysqli_sql_exception: Unknown column 'ciente.personal_id' in 'on clause' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:18:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ciente.personal_id' in 'on clause'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM (`cliente`, `cliente` AS `c`)
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ciente.personal_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM (`cliente`, `cliente` AS `c`)
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM (`cliente`, `cliente` AS `c`)
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM (`cliente`, `cliente` AS `c`)
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:18:49 --> [Caused by] mysqli_sql_exception: Unknown column 'ciente.personal_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM (`cliente`, `cliente` AS `c`)
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM (`cliente`, `cliente` AS `c`)
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM (`cliente`, `cliente` AS `c`)
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome` AS `cliente_nome`, `c`.`CPF`, `c`.`email`, `p`.`nome` AS `nome_personal`
FROM (`cliente`, `cliente` AS `c`)
INNER JOIN `funcionarios` AS `p` ON `ciente`.`personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:19:03 --> BadMethodCallException: Call to undefined method App\Models\ClienteModel::group_by
in SYSTEMPATH\Model.php on line 874.
 1 APPPATH\Controllers\Cliente.php(64): CodeIgniter\Model->__call('group_by', [...])
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:24:27 --> mysqli_sql_exception: Not unique table/alias: 'cliente' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM (...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM (...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM (...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM (...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(67): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:24:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Not unique table/alias: 'cliente'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM (`cliente`, `cliente`)
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(67): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:24:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Not unique table/alias: 'cliente'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM (`cliente`, `cliente`)
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM (`cliente`, `cliente`)
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM (`cliente`, `cliente`)
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(67): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:24:27 --> [Caused by] mysqli_sql_exception: Not unique table/alias: 'cliente'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM (`cliente`, `cliente`)
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM (`cliente`, `cliente`)
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM (`cliente`, `cliente`)
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM (`cliente`, `cliente`)
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(67): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:25:12 --> mysqli_sql_exception: Column 'id' in field list is ambiguous in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`, `n...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `n...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `n...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:25:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome` AS `nome_personal`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:25:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome` AS `nome_personal`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome` AS `nome_personal`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome` AS `nome_personal`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:25:12 --> [Caused by] mysqli_sql_exception: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome` AS `nome_personal`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome` AS `nome_personal`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome` AS `nome_personal`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome` AS `nome_personal`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:25:25 --> mysqli_sql_exception: Column 'id' in field list is ambiguous in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`, `n...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `n...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `n...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:25:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:25:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:25:25 --> [Caused by] mysqli_sql_exception: Column 'id' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:25:35 --> mysqli_sql_exception: Column 'nome' in field list is ambiguous in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `nome`, ...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nome`, ...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nome`, ...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `nome`, ...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:25:35 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'nome' in field list is ambiguous
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:25:35 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'nome' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:25:35 --> [Caused by] mysqli_sql_exception: Column 'nome' in field list is ambiguous
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `nome`, `email`, `CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:26:46 --> mysqli_sql_exception: Unknown column 'cliente' in 'field list' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `cliente...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:26:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'cliente' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:26:46 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'cliente' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:26:46 --> [Caused by] mysqli_sql_exception: Unknown column 'cliente' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:27:04 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`c`' at line 3 in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `cliente...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:27:04 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`c`' at line 3
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:27:04 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`c`' at line 3
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:27:04 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`c`' at line 3
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `cliente` as `c`, `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:27:11 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`c`' at line 3 in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:27:11 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`c`' at line 3
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:27:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`c`' at line 3
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:27:11 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '`c`' at line 3
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `cliente``as` `c`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:30:20 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`' at line 1 in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT (c.id, `...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT (c.id, `...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT (c.id, `...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT (c.id, `...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:30:20 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`' at line 1
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT (c.id, `c`.`nome`, `c`.`email`, `c`.`CPF`, p.nome) cliente as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:30:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT (c.id, `c`.`nome`, `c`.`email`, `c`.`CPF`, p.nome) cliente as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT (c.id, `c`.`nome`, `c`.`email`, `c`.`CPF`, p.nome) cliente as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT (c.id, `c`.`nome`, `c`.`email`, `c`.`CPF`, p.nome) cliente as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:30:20 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`' at line 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT (c.id, `c`.`nome`, `c`.`email`, `c`.`CPF`, p.nome) cliente as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT (c.id, `c`.`nome`, `c`.`email`, `c`.`CPF`, p.nome) cliente as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT (c.id, `c`.`nome`, `c`.`email`, `c`.`CPF`, p.nome) cliente as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT (c.id, `c`.`nome`, `c`.`email`, `c`.`CPF`, p.nome) cliente as c
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
ERROR - 2024-04-27 01:31:20 --> mysqli_sql_exception: Unknown column 'c.id' in 'field list' in C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`...', 0)
#1 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`...')
#2 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`...')
#3 C:\wamp64\www\sites\Projeto1\Back-end\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`...', Array, false)
#4 C:\wamp64\www\sites\Projeto1\Back-end\system\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\sites\Projeto1\Back-end\app\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', Array)
#6 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
#7 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
#8 C:\wamp64\www\sites\Projeto1\Back-end\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\sites\Projeto1\Back-end\public\index.php(85): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-04-27 01:31:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`
GROUP BY `c`.`id`', [], false)
 2 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:31:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`
GROUP BY `c`.`id`')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`
GROUP BY `c`.`id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`
GROUP BY `c`.`id`', [], false)
 4 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-27 01:31:20 --> [Caused by] mysqli_sql_exception: Unknown column 'c.id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`
GROUP BY `c`.`id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`
GROUP BY `c`.`id`')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`
GROUP BY `c`.`id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `c`.`id`, `c`.`nome`, `c`.`email`, `c`.`CPF`, `p`.`nome`
FROM `cliente`
JOIN `funcionarios` AS `p` ON `personal_id` = `p`.`id`
GROUP BY `c`.`id`', [], false)
 5 SYSTEMPATH\Model.php(872): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Controllers\Cliente.php(66): CodeIgniter\Model->__call('get', [])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Cliente->pesquisar()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Cliente))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
